shared/dataRepository.ts

```diff
@@ -48,8 +50,11 @@ const generateFilePath = (time: NzTimeIso) => {
   return `${CONSTANTS.dataFolder}/${date}.txt`;
 };
 
-const getLatestVaccineDataFilePath = async () => {
+const getVaccineDataFilePath = async (period: "latest" | "yesterday") => {
   const files = await fs.readdir(CONSTANTS.dataFolder);
-  const latestDataFile = files.sort()[files.length - 1];
-  return `${CONSTANTS.dataFolder}/${latestDataFile}`;
+  const filePath =
+    period === "latest"
+      ? files.sort()[files.length - 1]
+      : files.sort()[files.length - 2];
```

Comment:
Minor: Surely now this works 😅 

```suggestion
  const [latestPath, yesterdayPath] = files.sort().reverse();
  const filePath = period === "latest" ? latestPath : yesterdayPath;
```

Reply:
Lol of course!